### PR TITLE
Blog post : Retest add-on

### PR DESCRIPTION
**Pull request by pranavsaxena17**
Added a blog post for the retest add-on.

### CODE REVIEW

File: site/content/blog/2021-08-23-retest-with-zap/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Each selected alert has a status column in the table, which displays the current status of the alert, which is one out of 'Not Verified', 'Present' and 'Absent'.
+
+The user can click on the 'Create' button, which creates the plan and loads it in the automation panel. From here, the user can reorder/modify the jobs as required and also run the plan. More information on how the Automation Framework GUI works is given [here](https://www.zaproxy.org/docs/desktop/addons/automation-framework/gui/)
+Since a retest plan can be run from the command line too, it is also easy to integrate it within a CI/CD pipeline.
+
+## Retest Result 
+![Output Tab](images/result.png)
+
+The user can see the result of the running the retest plan in the Output tab of the automation panel. The 'Pass'/'Failed' status for an alert test depends on whether that alert was being checked for its absence or for its presence. This value can be configured by changing the 'Action' field
```
Comment (thc202) [inline site/content/blog/2021-08-23-retest-with-zap/index.md:39]: `result of the` → `result of`

File: site/content/blog/2021-08-23-retest-with-zap/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+of an alert via the Edit alert functionality described above.
+
+## The Retest API
+The add-on also has an API, which currently supports only one 'retest' action. The user can supply a list of comma separated alert ids for retesting, and would receive a response containing the 'Present'/'Absent' status of an alert along with the values of all the fields describing that alert.
```
Comment (thc202) [inline site/content/blog/2021-08-23-retest-with-zap/index.md:43]: IDs

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: site/content/blog/2021-08-23-retest-with-zap/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: "Retesting alerts with OWASP ZAP"
+description: "Introducing the Retest add-on for ZAP"
+summary: "An overview of the features of the Retest add-on for OWASP ZAP. This add-on allows you to retest for previously generated alerts."
+type: post
+tags:
+- blog
+date: "2021-08-23"
+addSocialPreview: true
```
Comment (psiinon) [inline site/content/blog/2021-08-23-retest-with-zap/index.md:9]: full url of one of the images?

**Review by psiinon — APPROVED** [review]

**Pull request merged**